src/Framework/Sdk/SdkLogger.cs

```diff
@@ -17,5 +19,32 @@ public abstract class SdkLogger
         /// <param name="message">Message string.</param>
         /// <param name="messageImportance">Optional message importances. Default to low.</param>
         public abstract void LogMessage(string message, MessageImportance messageImportance = MessageImportance.Low);
+
+        /// <summary>
+        /// Logs that an event.
+        /// </summary>
+        /// <param name="message">A message to associate with the event.</param>
+        public virtual void LogEvent(string message)
```

Comment:
Small concern: there are now two meanings of `Log` in this class:

1. Log a message that will show up in the MSBuild log.
2. Emit an ETW event if configured.

Is there a better name for the latter?

Reply:
I could use the same naming as ETW itself:
* `WriteEvent()`
* `WriteEventStart()`
* `WriteEventStop()`

What do you think?

Reply:
I like that better.